D-OS Save Editor/savegame/Meta.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace D_OS_Save_Editor
+{
+    public class Meta
+    {
+        public enum DifficultyEnum
+        {
+            Story = 0,
+            Classic,
+            Tactician,
+            Honour
+        }
+        public enum SaveGameTypeEnum
+        {
+            Manual = 0,
+            QuickSave,
+            AutoSave,
+            HonourSave
+        }
+        private string _level;
+        private int _difficulty;
+        private List<string> _gameVersion;
+        private int _saveGameType;
+        private string _seed;
+        private string _saveTime;
+        private List<string> _modsName;
+
+        //<node id="MetaData">
+
+        //<attribute id = "Level" value="Cyseal" type="22" />
+        /// <summary>
+        /// Game map/level
+        /// </summary>
+        public string Level { get => _level; set => _level = value; }
+        /// <summary>
+        /// Difficulty Game
+        /// </summary>
+        public int Difficulty { get => _difficulty; set => _difficulty = value; }
+        /// <summary>
+        /// Full game version
+        /// </summary>
+        public List<string> GameVersion { get => _gameVersion; set => _gameVersion = value; }
+        /// <summary>
+        /// Date of savegame created
+        /// </summary>
+        public string SaveTime { get => _saveTime; set => _saveTime = value; }
+        /// <summary>
+        /// List of Mod's Name load in game
+        /// </summary>
+        public List<string> ModsName { get => _modsName; set => _modsName = value; }
+        //<attribute id="SaveGameType" value="2" type="27" />
+        /// <summary>
+        /// The type of SaveGame
+        /// </summary>
+        public int SaveGameType { get => _saveGameType; set => _saveGameType = value; }
+        public string Seed { get => _seed; set => _seed = value; }
+
```

Comment:
Can we not use [auto properties](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/auto-implemented-properties) here?

Reply:
Yes of course, when i wrote this code, I didn't make difference between property access with/without treatment. If I understand that you say, I can write this:
public int SaveGameType { get; set; }
public string Seed { get; set; }
Isn't it ?

Reply:
Yes! It is much easier to read and code will be much neater :)